src/dstr-binding/error/try-catch-init.template

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+path: language/statements/try/dstr-catch-init-
+name: try catch with initializer statement
+esid: sec-runtime-semantics-catchclauseevaluation
+features: [destructuring-binding, catch-initializer]
+info: |
+  https://github.com/tc39/ecma262/pull/1126
+
+  Catch : catch ( CatchParameter ) Block
+
+  CatchParameter[Yield, Await] :
+    FormalParameter[?Yield, ?Await]
+---*/
+
+assert.throws(/*{ error }*/, function() {
+    try {
+      throw undefined;
+    } catch (/*{ elems }*/ = /*{ vals }*/) {}
+  });
+  
```

Comment:
in the meanwhile, it would be nice if you could fix this EOF for consistency, please.